src/stories/select/Select.stories.ts

```diff
@@ -32,18 +32,7 @@ export const Default: Story = {
             { id: '1', node: 'Single' },
             { id: '2', node: 'EP' },
             { id: '3', node: 'Album (LP)' },
-            { id: '3', node: 'Album (LP)' },
-            { id: '3', node: 'Album (LP)' },
-            { id: '3', node: 'Album (LP)' },
-            { id: '3', node: 'Album (LP)' },
-            { id: '3', node: 'Album (LP)' },
-            { id: '3', node: 'Album (LP)' },
-            { id: '3', node: 'Album (LP)' },
-            { id: '3', node: 'Album (LP)' },
-            { id: '3', node: 'Album (LP)' },
-            { id: '3', node: 'Album (LP)' },
-            { id: '3', node: 'Album (LP)' },
-            { id: '3', node: 'Album (LP)' },
+            { id: '4', node: 'Compilation' },
```

Comment:
I agree with removing the repeated album LP (I think this was probably something I did in testing at some point and forgot to remove before pushing) but can we add a few more options here? would be nice to have a scrollable example